### ISSUE TITLE
onnx batch_size为-1 静态维度问题

### ISSUE THREAD

**Issue by Affectionate-0** [PaddlePaddle/Paddle2ONNX#901]
def forward(self, input_ids, token_type_ids, pos_ids, att_mask):
    sequence_output, pooled_output = self.encoder(
        input_ids=input_ids,
        token_type_ids=token_type_ids,
        position_ids=pos_ids,
        attention_mask=att_mask)
    sequence_output = paddle.reshape(sequence_output,[-1,512,12,26])
    eca_output = self.eca(sequence_output) 
    eca_output = paddle.reshape(eca_output,[-1,512,312])
    start_logits = self.linear_start(eca_output )
    start_logits = paddle.squeeze(start_logits, -1)
    start_prob = self.sigmoid(start_logits)
    end_logits = self.linear_end(eca_output )
    end_logits = paddle.squeeze(end_logits, -1)
    end_prob = self.sigmoid(end_logits)
    return start_prob, end_prob

针对UIE的改动，模型转静态图正常，但是在加载静态图infer时出现了以下报错（楼上-1为静态维度的问题导致不能广播）：
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Non-zero status code returned while running Expand node. Name:'p2o.Expand.1' Status Message: p2o.Expand.1: right operand cannot broadcast on dim 0 LeftShape: {1,512,1,1}, RightShape: {-1,512,12,26}

更新了 onnxsim --> Successfully installed commonmark-0.9.1 onnx-simplifier-0.4.8 rich-12.5.1
通过onnxsim保存模型之后加载还是报上述错误！
期待回复！

**Comment by yeliang2258** [comment]
这个看起来像是broadcast的两个输入顺序反了的问题，试一下paddleinference是否可以正常推理？如果可以正常推理的话，麻烦提供一下你的模型，我们来看看是什么问题。

**Comment by Affectionate-0** [comment]
> 这个看起来像是broadcast的两个输入顺序反了的问题，试一下paddleinference是否可以正常推理？如果可以正常推理的话，麻烦提供一下你的模型，我们来看看是什么问题。


正常加载checkpoint可以预测
    tokenizer = AutoTokenizer.from_pretrained(args.model_path)
    paddle.set_device(args.device)
    model = UIE.from_pretrained(args.model_path)

就在转了静态图之后infer过程中    报了-1不能广播的错

**Comment by Affectionate-0** [comment]
模型文件超过25M无法上传，我用什么方式给你。
